AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2701,3 +2701,4 @@ jianshu.com#$#body { overflow: auto !important; }
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
 instiz.net#$?#.backlogo { background-image: none !important; }
+||script.gmarket.co.kr/js/itemdweb/ItemBase.js
```

Comment:
Better to this file https://github.com/AdguardTeam/AdguardFilters/blob/10c4b3f8492fae170cb378183c9e1bd4364b2c6e/AnnoyancesFilter/sections/tweaks.txt#L1729

Reply:
#4519013